src/components/Tool/FixMailingAddresses/Contact.tsx

```diff
@@ -321,9 +322,6 @@ const Contact: React.FC<Props> = ({
                       <strong>{t('Source')}: </strong>
                     </Typography>
                   </Hidden>
-                  <Typography display="inline">
-                    {t('{{appName}}', { appName })}
-                  </Typography>
                 </Box>
```

Comment:
You've removed the source?

Reply:
Yes, it is not needed there.

Reply:
The dropdown will be filled in with values that existing addresses have, and they can choose which source to create the address in. So, if they already have a DonorHub and an MPDX address, both will appear in the dropdown to choose. This impacts the logic for `primary` when saving as well.

Reply:
I think this is what Bill is referring to with the dropdown. There are 2 MPDX values. I assume this is due to one being something else like `MPDX6` but then being changed to MPDX on the UI.

![Screenshot 2024-10-09 at 9 24 25 AM](https://github.com/user-attachments/assets/84a87109-0746-41ad-af4a-55a4e10c520c)

Reply:
> In my opinion, I think it is self explanatory that if they create the address in MPDX, then the source will be MPDX. Just like if they create a new contact or person or donation or anything

I think it's best if we make sure the user knows they are creating the address on MPDX. This will avoid confusion.

I'm open to changing how we tell the user a new address will be created on MPDX.

Reply:
Ok I will add the source to the Add Address button so it will look like: `Add Address (MPDX)`